cdap-data-fabric/src/main/java/io/cdap/cdap/store/StoreDefinition.java

```diff
@@ -179,6 +181,70 @@ public static void create(StructuredTableAdmin tableAdmin) throws IOException {
     }
   }
 
+  /**
+   * Schema for SourceControlMetadata table.
+   */
+  public static final class SourceControlMetadataStore {
+
+    public static final StructuredTableId SOURCE_CONTROL_METADATA = new StructuredTableId("source_control_metadata");
+
+    public static final String NAMESPACE_FIELD = "namespace";
+    public static final String TYPE_FIELD = "type";
+    public static final String NAME_FIELD = "name";
+    public static final String SPECIFICATION_HASH_FIELD = "specification_hash";
+    public static final String COMMIT_ID_FIELD = "commit_id";
+    public static final String LAST_MODIFIED_FIELD = "last_modified";
+    public static final String IS_SYNCED_FIELD = "is_synced";
+
+    public static final StructuredTableSpecification SOURCE_CONTROL_METADATA_TABLE_SPEC = new StructuredTableSpecification.Builder()
```

Comment:
Please fix checkstyle

Reply:
Done

Reply:
Edit: There are many checkstyle errors yet to be fixed